sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -93,8 +92,11 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
     }
   }
 
-  private def requireTableExists(db: String, table: String): Unit = {
-    withClient { getTable(db, table) }
+  /**
+   * Get the raw table metadata from hive metastore directly.
+   */
+  private def getRawTable(db: String, table: String): CatalogTable = withClient {
```

Comment:
This is from my original comment: https://github.com/apache/spark/pull/14750#issuecomment-245788615 

Personally, I think it is easier for the other coders to understand the differences between `getTable` and `client.getTable`